python-checks/src/test/resources/checks/cdk/s3BucketVersioning.py

```diff
@@ -0,0 +1,42 @@
+from aws_cdk import aws_s3 as s3
+
+a = int(10)
+
+bucket = s3.Bucket(self, "MyUnversionedBucket", versioned=False) # Noncompliant
+                                              # ^^^^^^^^^^^^^^^
+
+# Noncompliant@+1 {{Make sure using unversioned S3 bucket is safe here. Omitting 'versioned=True' disables S3 bucket versioning. Make sure it is safe here.}}
+bucket = s3.Bucket(self, "MyUnversionedBucket")
+#        ^^^^^^^^^
+
+bucket_versioning = False
```

Comment:
We should add a secondary location for the assignment. So it's more straightforward for users why we raise an issue on line 14.